Makefile

```diff
@@ -211,7 +211,13 @@ protobuild:
 .PHONY: protolint
 protolint:
 	@buf lint
-	@buf breaking --against 'https://github.com/hashicorp/boundary.git#branch=stable-website'
+	# First check all protos except controller/servers for WIRE_JSON compatibility
+	cd internal/proto && buf breaking --against 'https://github.com/hashicorp/boundary.git#branch=stable-website,subdir=internal/proto' \
+		--exclude-path=controller/servers \
+		--config '{"breaking":{"use":["WIRE_JSON"],"ignore_only":{"FIELD_NO_DELETE_UNLESS_NUMBER_RESERVED":["controller/api","controller/storage"]}},"version":"v1","name":"buf.build/hashicorp/boundary","deps":["buf.build/googleapis/googleapis","buf.build/grpc-ecosystem/grpc-gateway"]}'
```

Comment:
Can you update this to place the configs into separate files? I tested it out and this still works, so something like:

`internal/proto/buf.breaking.api.yaml`:
```yaml
breaking:
  use:
    - WIRE_JSON
  ignore_only:
    FIELD_NO_DELETE_UNLESS_NUMBER_RESERVED:
      - controller/api
      - controller/storage
version: v1
name: buf.build/hashicorp/boundary
deps:
  - buf.build/googleapis/googleapis
  - buf.build/grpc-ecosystem/grpc-gateway
```
`internal/proto/buf.breaking.grpc.yaml`
```yaml
breaking:
  use:
    - WIRE
  ignore_only:
    FIELD_NO_DELETE_UNLESS_NUMBER_RESERVED:
      - controller/api
      - controller/storage
version: v1
name: buf.build/hashicorp/boundary
deps:
  - buf.build/googleapis/googleapis
  - buf.build/grpc-ecosystem/grpc-gateway
```

Then you would need to update this to `--config=./buf.breaking.api.yaml` and the other to `--config=./buf.breaking.grpc.yaml`